src/test/java/com/wikia/webdriver/common/templates/NewTestTemplateCore.java

```diff
@@ -93,10 +94,11 @@ protected void logOutCustomDriver(WebDriver customDriver) {
   }
 
   protected void stopBrowser() {
-    /*if (NewDriverProvider.getMobileDriver() != null
-        && NewDriverProvider.getMobileDriver().getSessionId() != null) {
-      NewDriverProvider.getMobileDriver().quit();
-    }*/
+    /*
+     * if (NewDriverProvider.getMobileDriver() != null &&
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)